bundle/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -160,21 +160,21 @@ metadata:
       ]
     capabilities: Seamless Upgrades
     categories: Monitoring,Security
-    olm.skipRange: '>=0.1.17 <0.1.61'
+    olm.skipRange: '>=0.1.17 <1.0.0'
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-compliance
     operators.openshift.io/infrastructure-features: '["disconnected", "fips", "proxy-aware"]'
     operators.openshift.io/valid-subscription: '["OpenShift Kubernetes Engine", "OpenShift
       Container Platform", "OpenShift Platform Plus"]'
     operators.operatorframework.io/builder: operator-sdk-v1.19.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-    repository: https://github.com/ComplianceAsCode/compliance-operator
+    repository: https://github.com/openshift/compliance-operator
```

Comment:
For some reason this is getting overwritten - https://github.com/ComplianceAsCode/compliance-operator/pull/230/files

Reply:
This should be fixed when we rebase on https://github.com/ComplianceAsCode/compliance-operator/pull/275

Reply:
Ohh How did I missed this one